apps/hobbygroups/models.py

```diff
@@ -8,6 +8,7 @@ class Hobby(models.Model):
     description = models.TextField(max_length=250)
     image = models.ImageField(upload_to=settings.IMAGES_PATH)
     read_more_link = models.URLField(blank=True)
+    hobby_priority = models.IntegerField(default=0)
 
     def __str__(self):
-        return self.title
+        return (self.title + " [Priority: " + str(self.hobby_priority) + "]")
```

Comment:
Could use string formatting rather than concatenation for prettier code;
`return "{title} (Priority: {priority})".format(title=self.title, priority=self.hobby_priority)`